packages/twenty-front/src/App.tsx

```diff
@@ -120,7 +120,7 @@ export const App = () => {
                     <Routes>
                       <Route
                         path={SettingsPath.Developers}
```

Comment:
think we should update the `settingsPath.Developers` value
![image](https://github.com/twentyhq/twenty/assets/29927851/e53efeae-9777-43b4-8d04-b1d7831acf87)
Think you can just replace it by `""`. Update hrefs in `SettingsDevelopersApiKeyDetail` and `SettingsDevelopersApiKeysNew`